### PR TITLE
[Task-9][Frontend] As a user, I can logout

### LINKED ISSUE #9: [Task-9][Frontend] As a user, I can logout
## Why

Users can log out from the Navbar.
 
## Acceptance Criteria
- [ ] Create a navbar
- [ ] Log out button should be in the navbar

### PR DESCRIPTION
**Pull request by rafayet-monon**
Task Card - https://github.com/rafayet-monon/elixir_search_extractor/projects/1#card-61390221

## What happened 👀

Users can log out from the Navbar.
 
## Insight 📝

- [x] Create a navbar
- [x] Log out button should be in the navbar
 
## Proof Of Work 📹

image

### CODE REVIEW

File: assets/css/layouts/default.scss [hunk 1]
```diff
@@ -3,11 +3,11 @@
     display: flex;
     flex-flow: column;
     height: 100vh;
+    font-family: 'Roboto', sans-serif;
```
Comment (rosle) [inline assets/css/layouts/default.scss]: Is it possible to config this in the bootstrap variables 🤔

File: assets/css/_variables.scss [hunk 2]
```diff
@@ -1,5 +1,6 @@
 $font-size: (
   sm: .875rem,   // 14px
+  md: 1rem,   // 14px
```
Comment (rosle) [inline assets/css/_variables.scss]:
```suggestion
  md: 1rem,   // 16px
```

Comment (rafayet-monon) [reply assets/css/_variables.scss]: @rosle Fixed issues in [6c62c24](https://github.com/rafayet-monon/elixir_search_extractor/pull/30/commits/6c62c2457e71b799e3948236c09ce4a9fefa47fe)

**Review by rosle — APPROVED** [review]
👍

**Review by hanam1ni — APPROVED** [review]

**Pull request merged**